src/lib/utils/theme/DefaultTheme.ts

```diff
@@ -9,6 +9,18 @@ const defaultMuiTheme: ThemeOptions = {
   palette: palette,
   components: {
     MuiLink: linkTheme,
+    MuiButton: {
+      styleOverrides: {
+        root: {
+          disableRipple: true,
```

Comment:
I think `disableRipple` should be in `defaultProps` rather than style overrides. Prefer theming in `defaultProps` when possible because it's easier to override in special cases.